code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -782,7 +782,9 @@ GLOBAL_LIST_EMPTY(features_by_species)
 							else
 								accessory_overlay.color = source.dna.features["mcolor"]
 						if(HAIR_COLOR)
-							if(hair_color == "mutcolor")
+							if (source.hairstyle == "Bald" && istype(accessory, /datum/sprite_accessory/ears/cat))
```

Comment:
This is inconsistent with the changes to the other file, which do not exclusively target cat ears. It should also have lower priority than the "mutcolor"/"fixedmutcolor" checks below.

Reply:
i changed the priority, removed the check for cat ears
this still makes sense to me in that, if you have a mutation that uses your hair color and you have no hair, to just stick with skintone
besides, currently the only things that use the HAIR_COLOR trait are cat ears and cat tails
i tried moving it out to felinids.dm with force_colour but that didn't seem to work

Reply:
Fox ears and tails also use hair color iirc.